packages/help-center/src/components/help-center-more-resources.tsx

```diff
@@ -62,7 +52,6 @@ export const HelpCenterMoreResources = () => {
 
 	const trackMoreResourcesButtonClick = ( resource: string ) => {
 		recordTracksEvent( 'calypso_help_moreresources_click', {
-			is_business_or_ecommerce_plan_user: isBusinessOrEcomPlanUser,
```

Comment:
Confirm that it's OK to remove this prop. It's expensive to calculate its value and it's from 3 years ago.